### PR TITLE
Update enhanced-filtering-for-connectors.md

### PR DESCRIPTION
**Pull request by SignorelliDenis**
1: Mentioned that EFUsers value must match the targetAddress of the MailUser object in hybrid mail flow (related to the issue: https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/1742). I came across this situation a couple of times where customers complain that Skip Listing was not working when in fact they just didn't add the targetAddress mail.onmicrosoft.com. IMO would be nice to mention the in this situation mail.onmicrosoft.com is the right value to add. 

2: Changed the IP examples from private IP to Public IP.  (related to the issue: https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/1960)

3: Mentioned that we can’t add private IP to the Skip Listing. Private IP is automatically skipped.

Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/1742

Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/1960

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mail-flow-best-practices/use-connectors-to-configure-mail-flow/enhanced-filtering-for-connectors.md [hunk 1]
```diff
@@ -105,6 +105,8 @@ Using the previous example, you would configure the IP address of the third-part
 
        - This option is only affective on the actual email address that you specify. For example, if a user has five email addresses associated with their mailbox (also known as _proxy addresses_), you'll need to specify all five of their email addresses here. Otherwise, messages that are sent to the four other email addresses will go through normal filtering.
 
+        - For hybrid environment which the inbound mail flow is through Exchange On-Premises, you'll need to specify the targetAddress of the MailUser object. E.g.: michelle@contoso.mail.onmicrosoft.com
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/use-connectors-to-configure-mail-flow/enhanced-filtering-for-connectors.md:108]:
```suggestion
        - For a hybrid environment in which the inbound mail flow is through on-premises Exchange, you'll need to specify the targetAddress of the MailUser object, for example, _michelle@contoso.mail.onmicrosoft.com_.
```

File: Exchange/ExchangeOnline/mail-flow-best-practices/use-connectors-to-configure-mail-flow/enhanced-filtering-for-connectors.md [hunk 2]
```diff
@@ -146,7 +148,7 @@ Set-InboundConnector -Identity "From Anti-spam Service" -EFSkipLastIP $true -EFU
 ```
 
 > [!IMPORTANT]
-> Entering the IP addresses of Office 365 is not supported. Do not use this feature to compensate for issues introduced by unsupported email routing paths. Use caution and limit the IP ranges to only the mail systems that will handle your own organization's messages prior to Office 365.
+> Entering the IP addresses of Office 365 is not supported. Do not use this feature to compensate for issues introduced by unsupported email routing paths. Use caution and limit the IP ranges to only the mail systems that will handle your own organization's messages prior to Office 365. Entering private IP address is not supported as well. Enhanced Filtering automatically detect and skip private IP addresses.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/use-connectors-to-configure-mail-flow/enhanced-filtering-for-connectors.md:151]:
```suggestion
> Entering the IP addresses of Office 365 is not supported. Do not use this feature to compensate for issues introduced by unsupported email routing paths. Use caution and limit the IP ranges to only the mail systems that will handle your own organization's messages prior to Office 365. Entering a private IP address is also not supported. Enhanced Filtering automatically detects and skips private IP addresses.
```

**Comment by mattpennathe3rd** [comment]
@SignorelliDenis Molte grazie! Apprezzo l'auto.

This will be published in the near future.

**Pull request merged**